tests/make/yaml_lint_failure.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+TEST_DIR="$(pwd)/yaml_lint_failure"
+
+# generate a test terraform project with a nested project
+function setup() {
+rm -rf "$TEST_DIR"
+working_dirs=("$TEST_DIR" "$TEST_DIR/nested")
+for working_dir in "${working_dirs[@]}"
+do
+
+  mkdir -p "$working_dir"
+  cat > "$working_dir/test.yml" <<"EOF"
+foo: "bar
```

Comment:
This seems like it can just be a simple echo `"foo: \"bar\"" > "$working_dir/test.yml"` instead of the whole EOF thing

Reply:
true. personally i think the tests are a little easier to read by keeping it a heredoc. you don't have to do any in-head string conversions or char escaping in order to grok the end result

Reply:
So you are actually missing a quote after `bar` so I am curious if these extra quotes are needed to begin with? Can I just do `echo "foo: bar" > $working_dir/test.yml`

Reply:
huh. according to yamllint this is valid yaml `foo: bar"` but this is not `foo: "bar`.. updateda